### DISCUSSION TITLE
Qtable add q-btn-dropdown

### DISCUSSION THREAD

**Discussion by BalazsMakrai** [quasarframework/quasar#14838] [General - Components / Directives / etc] [answered]
I want to add a dropdown button to my Qtable. 
Here is the solution that I want to achieve:

![dsdsdsd](https://user-images.githubusercontent.com/64318333/200289504-a39a53f5-5eaf-4b4f-8739-3089f81e922e.JPG)

Is there any way to do it?

**Comment by suleimanas** [comment, marked as answer]
use slots: https://quasar.dev/vue-components/table#body-slots

add QSelects - you already have the data so you can group the data for qselect options
